workshops/util.py

```diff
@@ -46,22 +43,9 @@ def upload_person_task_csv(uploaded_file, encoding=None):
 
     Also return a list of fields from Person.PERSON_UPLOAD_FIELDS for which
     no data was given.
-
-    :param string encoding: encoding used to encode incoming file. Defaults to
-                            ``django.conf.settings.DEFAULT_CHARSET``
     """
     persons_tasks = []
 
-    if not encoding:
-        encoding = settings.DEFAULT_CHARSET
-
-    # we provide uploaded_file as StringIO in our tests (test_util.py)
-    if not issubclass(StringIO, uploaded_file.__class__):
-        # Django provides uploaded files as byte file objects.  We need to wrap
-        # `uploaded_file` and provide it as a text file with specific encoding.
-        # This way we force users to upload UTF-8 encoded files.
-        uploaded_file = TextIOWrapper(uploaded_file.file, encoding=encoding)
-
     reader = csv.DictReader(uploaded_file)
```

Comment:
> No need to apologize ;). Thanks for pushing back on anything that isn't clear, since I certainly make stupid mistakes that I'd rather have caught and fixed during the review phase. Just because this one thing seems ok, doesn't mean I'll be right the next time I do something that you feel is suspicious ;).

Yup, I'm used to making mistakes :) (therefore I'm used to apologizing for my mistakes :) )

Anyway, that's what code review is, right? We take a critical look at each others code.